### ISSUE TITLE
One of the useful CDL featues was considered a bug and was "fixed"

### ISSUE THREAD

**Issue by cyneprepou4uk** [TASEmulators/fceux#771]
65f4a482d560cf9e1e04b4ee4601245da7b74695

While I realize the reason why this can be considered a bug, in fact it helps to see code flow in Hex Editor and Debugger, and warn the romhacker that some code could be jumping there.

Also I use this Data flag in my CDL-based disasembler, where it helps to quickly label cross-bank jumps. Because sometimes several banks can have code at the same jump location, and you need to figure out which bank it is. But if only 1 bank has this flag set, it's easy.

Basically this CDL flag marks 4C jumps the same way as bit4 marks 6C jumps. And if no 4C jumps found jumping at some code location, but the flag is still set at this location, this could lead to finding possible a bug in the game, like some code is reading what it's not supposed to.

![image](https://github.com/user-attachments/assets/041dce01-e6af-42f1-ae2a-08b59a173768)

@parasyte please revert this.

**Comment by parasyte** [comment]
The last discussion on this was in #755. We can't break PRG ROM writes. Reverting would not be acceptable, either.

I have a private branch where I was experimenting with adding more context to CDL files by extending it with a second overlay with 32-bits of context per byte. It adds a proper "branch target" flag and some others that are used to guide the disassembler. Mostly for instruction disambiguation and labels.

I'm sorry fixing the bug affected your use case ([Hyrum's Law](https://www.hyrumslaw.com/)) but there are better ways. We should move forward, not back.
